### PR TITLE
[Refactor] Fix some compaction misleading log

### PR DESCRIPTION
**Pull request by meegoo**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [ ] enhancement
- [x] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by meegoo** [comment]
@mergify rebase

**Comment by meegoo** [comment]
run starrocks_be_unittest

File: be/src/storage/compaction_scheduler.cpp [hunk 1]
```diff
@@ -120,13 +120,13 @@ bool CompactionScheduler::_can_do_compaction_task(Tablet* tablet, CompactionTask
...
             return false;
         }
-        last_failure_ms = tablet->last_base_compaction_failure_time();
+        last_failure_ts = tablet->last_base_compaction_failure_time();
     }
-    int64_t now_ms = UnixMillis();
-    if (now_ms - last_failure_ms <= config::min_compaction_failure_interval_sec * 1000) {
-        LOG(INFO) << "Too often to schedule compaction, skip it."
+    if (UnixMillis() - last_failure_ts <= config::min_compaction_failure_interval_sec * 1000) {
+        LOG(INFO) << "Too often to schedule failure compaction, skip it."
```
Comment (ABingHuang) [inline be/src/storage/compaction_scheduler.cpp:126]: In some case, this log may be noisy. better to change LOG(INFO) to VLOG(1)?

**Review by ABingHuang — APPROVED** [review]
LGTM

File: be/src/storage/compaction_scheduler.cpp [hunk 2]
```diff
@@ -120,13 +120,13 @@ bool CompactionScheduler::_can_do_compaction_task(Tablet* tablet, CompactionTask
                       << ", running num:" << num;
             return false;
         }
-        last_failure_ms = tablet->last_base_compaction_failure_time();
+        last_failure_ts = tablet->last_base_compaction_failure_time();
     }
-    int64_t now_ms = UnixMillis();
-    if (now_ms - last_failure_ms <= config::min_compaction_failure_interval_sec * 1000) {
-        LOG(INFO) << "Too often to schedule compaction, skip it."
+    if (UnixMillis() - last_failure_ts <= config::min_compaction_failure_interval_sec * 1000) {
```
Comment (jaogoy) [inline be/src/storage/compaction_scheduler.cpp:125]: UnixMillis() returns milliseconds, why did you rename _ms to _ts?

Comment (meegoo) [reply be/src/storage/compaction_scheduler.cpp:125]: ts short for timestamp

**Review by chaoyli — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by meegoo** [comment]
run starrocks_admit_test
